domain/application.ts

```diff
@@ -11,3 +11,9 @@ export interface Application {
   status: string;
   createdAt: string;
 }
+
+export interface ApplicationWithError extends Application {
```

Comment:
Is this an axios thing? we should get a response or an exception from the API. I wouldn't expect to extend the application here.